### DISCUSSION TITLE
虽然有点晚了

### DISCUSSION THREAD

**Discussion by BTMuli** [Arondight/Adachi-BOT#897] [General]
发现了一篇爆料专栏：
https://www.bilibili.com/read/cv17975758/
https://www.bilibili.com/read/cv17975769/

起码按照目前官方透露出来的消息来看，内容可信度还是挺高的。

里面提到了全新角色天赋跟武器突破材料的名称：
`诤言` `笃行` `巧思` `绿洲花园的恩惠` `烈日权威的余光` `谧林涓露的铁符`
不过角色天赋对应上跟官方的对不上
![image](https://user-images.githubusercontent.com/72692909/185786057-b86bad2d-ee39-4d50-ad2a-57f20a90471a.png)

**Comment by mark9804** [comment]
emmm，确实可信度很高，感觉可以用了
